meta/flag-references/en.md

```diff
@@ -0,0 +1,237 @@
+[flag___]: /wiki/shared/flag/__.gif
+[flag_AD]: /wiki/shared/flag/AD.gif "Andorra"
+[flag_AE]: /wiki/shared/flag/AE.gif "United Arab Emirates"
+[flag_AF]: /wiki/shared/flag/AF.gif "Afghanistan"
+[flag_AG]: /wiki/shared/flag/AG.gif "Antigua and Barbuda"
+[flag_AI]: /wiki/shared/flag/AI.gif "Anguilla"
+[flag_AL]: /wiki/shared/flag/AL.gif "Albania"
+[flag_AM]: /wiki/shared/flag/AM.gif "Armenia"
+[flag_AN]: /wiki/shared/flag/AN.gif "Netherlands Antilles"
+[flag_AO]: /wiki/shared/flag/AO.gif "Angola"
+[flag_AR]: /wiki/shared/flag/AR.gif "Argentina"
+[flag_AS]: /wiki/shared/flag/AS.gif "American Samoa"
+[flag_AT]: /wiki/shared/flag/AT.gif "Austria"
+[flag_AU]: /wiki/shared/flag/AU.gif "Australia"
+[flag_AW]: /wiki/shared/flag/AW.gif "Aruba"
+[flag_AX]: /wiki/shared/flag/AX.gif "Aland Islands"
+[flag_AZ]: /wiki/shared/flag/AZ.gif "Azerbaijan"
+[flag_BA]: /wiki/shared/flag/BA.gif "Bosnia and Herzegovina"
+[flag_BB]: /wiki/shared/flag/BB.gif "Barbados"
+[flag_BD]: /wiki/shared/flag/BD.gif "Bangladesh"
+[flag_BE]: /wiki/shared/flag/BE.gif "Belgium"
+[flag_BF]: /wiki/shared/flag/BF.gif "Burkina Faso"
+[flag_BG]: /wiki/shared/flag/BG.gif "Bulgaria"
+[flag_BH]: /wiki/shared/flag/BH.gif "Bahrain"
+[flag_BI]: /wiki/shared/flag/BI.gif "Burundi"
+[flag_BJ]: /wiki/shared/flag/BJ.gif "Benin"
+[flag_BL]: /wiki/shared/flag/BL.gif "Saint Barthelemy"
+[flag_BM]: /wiki/shared/flag/BM.gif "Bermuda"
+[flag_BN]: /wiki/shared/flag/BN.gif "Brunei"
+[flag_BO]: /wiki/shared/flag/BO.gif "Bolivia"
+[flag_BR]: /wiki/shared/flag/BR.gif "Brazil"
+[flag_BS]: /wiki/shared/flag/BS.gif "Bahamas"
+[flag_BT]: /wiki/shared/flag/BT.gif "Bhutan"
+[flag_BW]: /wiki/shared/flag/BW.gif "Botswana"
+[flag_BY]: /wiki/shared/flag/BY.gif "Belarus"
+[flag_BZ]: /wiki/shared/flag/BZ.gif "Belize"
+[flag_CA]: /wiki/shared/flag/CA.gif "Canada"
+[flag_CD]: /wiki/shared/flag/CD.gif "Congo, The Democratic Republic of the"
+[flag_CF]: /wiki/shared/flag/CF.gif "Central African Republic"
+[flag_CG]: /wiki/shared/flag/CG.gif "Congo"
+[flag_CH]: /wiki/shared/flag/CH.gif "Switzerland"
+[flag_CI]: /wiki/shared/flag/CI.gif "Cote D'Ivoire"
+[flag_CL]: /wiki/shared/flag/CL.gif "Chile"
+[flag_CM]: /wiki/shared/flag/CM.gif "Cameroon"
+[flag_CN]: /wiki/shared/flag/CN.gif "China"
+[flag_CO]: /wiki/shared/flag/CO.gif "Colombia"
+[flag_CR]: /wiki/shared/flag/CR.gif "Costa Rica"
+[flag_CU]: /wiki/shared/flag/CU.gif "Cuba"
+[flag_CV]: /wiki/shared/flag/CV.gif "Cabo Verde"
+[flag_CX]: /wiki/shared/flag/CX.gif "Christmas Island"
+[flag_CY]: /wiki/shared/flag/CY.gif "Cyprus"
+[flag_CZ]: /wiki/shared/flag/CZ.gif "Czech Republic"
```

Comment:
![image](https://user-images.githubusercontent.com/19843905/71453303-dc1bc900-278a-11ea-9150-0bc22359463a.png)

is now Czechia instead

Reply:
the countries are taken from osu-web, this one wasn't updated on purpose because its long-form name can still be called czech republic in english or something (idk just google this i remember looking it up before)